ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-multicast.j2

```diff
@@ -24,6 +24,19 @@
 {%     elif router_multicast.ipv4.software_forwarding is arista.avd.defined('sfe') %}
 - Software forwarding by the Software Forwarding Engine (SFE)
 {%     endif %}
+{%     if router_multicast.rpf.routes is arista.avd.defined %}
```

Comment:
This is causing the document to not print the table. This has to be ```router_multicast.ipv4.rpf.routes```

Reply:
Yep, actually just caught this while fixing the table edits you suggested with nexthop.  Just refreshing the facts now.

Thanks!